test/packsUpdated.test.js

```diff
@@ -0,0 +1,39 @@
+import {expect, jest, test} from '@jest/globals';
```

Comment:
This test replaces the old `package.json` script:

```
"test:packs": "npm run db:pack && git diff --no-patch --exit-code packs/weaponfx.db && echo 'Pack test passed!'"
```

It ensures that the compiled packs and the source of those packs are equal. This can be removed if we decide to stop storing the compiled packs in the repository.